### ISSUE TITLE
com.vkontakte.android - calling is not working

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/CoreLibs#1095]
> С включенным адгуард не происходит соединение с абонентом

> Можете попробовать тут https://vk.com/app6975290 нажать старт в 10:00, звонок от героев. Отвечает бот сразу. Логи скинул на почту в support

Original issue: https://github.com/AdguardTeam/AdguardFilters/issues/39247

**Comment by Alex-302** [comment]
У меня не повторилось.

**Comment by krotelot** [comment]
После нажатия, перезванивает бот, которому нужно ответить. После ответа у вас появляется видео?

**Comment by krotelot** [comment]
Можете попробовать еще знакомому позвонить, соеденилось?

**Comment by TheHasagi** [comment]
@krotelot 

Попробуйте отключить HTTPS фильтрацию для приложения.

**Comment by krotelot** [comment]
Было отключено

**Comment by TheHasagi** [comment]
Смотрю в ваш лог и вижу, что не отключена.

**Comment by TheHasagi** [comment]
AdGuard - Менеджер приложений - vkontatkte - Фильтрация HTTPS - откл.

**Comment by krotelot** [comment]
Да, была включена, потом отключил, Думал если включу заработает, до включения тоже не работало.

**Comment by krotelot** [comment]
Вот думаю, переустановка Adguard решит проблему?

**Comment by krotelot** [comment]
Переустановка не помогла, что-то блокирует соединение. 
Само приложение Вконтакте:
Если убрать блокировку рекламы, не помогает. Если убрать защиту AdGuard, то помогает.

**Comment by krotelot** [comment]
Если нужно, могу ещё раз логи собрать, не проблема.

**Comment by Alex-302** [comment]
Логи в тикете 1796261
